src/renderer/dx12.rs

```diff
@@ -592,28 +617,15 @@ impl RenderEngine {
         io.nav_active = true;
         io.nav_visible = true;
 
+        set_io_configs(io);
+
         // Map key indices to the virtual key codes
-        io[Key::Tab] = VK_TAB.0 as _;
-        io[Key::LeftArrow] = VK_LEFT.0 as _;
-        io[Key::RightArrow] = VK_RIGHT.0 as _;
-        io[Key::UpArrow] = VK_UP.0 as _;
-        io[Key::DownArrow] = VK_DOWN.0 as _;
-        io[Key::PageUp] = VK_PRIOR.0 as _;
-        io[Key::PageDown] = VK_NEXT.0 as _;
-        io[Key::Home] = VK_HOME.0 as _;
-        io[Key::End] = VK_END.0 as _;
-        io[Key::Insert] = VK_INSERT.0 as _;
-        io[Key::Delete] = VK_DELETE.0 as _;
-        io[Key::Backspace] = VK_BACK.0 as _;
-        io[Key::Space] = VK_SPACE.0 as _;
-        io[Key::Enter] = VK_RETURN.0 as _;
-        io[Key::Escape] = VK_ESCAPE.0 as _;
-        io[Key::A] = VK_A.0 as _;
-        io[Key::C] = VK_C.0 as _;
-        io[Key::V] = VK_V.0 as _;
-        io[Key::X] = VK_X.0 as _;
-        io[Key::Y] = VK_Y.0 as _;
-        io[Key::Z] = VK_Z.0 as _;
+        for i in keys::KEYS {
+            io[i.0] = i.1 .0 as _;
+        }
```

Comment:
This is the part that we can keep, and it's a very good idea.

Reply:
Great I think it make the code cleaner